### PR TITLE
add specLevel test

### PR DESCRIPTION
**Pull request by KewinTaoAnh**

### CODE REVIEW

File: src/test/java/com/kainos/ea/dao/SpecificationLevelTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    int id = 1;
+    specificationLevel = new SpecificationLevel();
+    JobRole result;
+
+    //When
+    result = specificationLevel.getJobRole(id);
+
+    //Then
+    assertThat(result).isNotNull();
+    assertFalse(result.getSpecification() == null);
```
Comment (hellofadude) [inline src/test/java/com/kainos/ea/dao/SpecificationLevelTest.java:28]: Can you assert on the exact Id returned for instance assertThat(result).contains(specification)

Comment (KewinTaoAnh) [reply src/test/java/com/kainos/ea/dao/SpecificationLevelTest.java:31]: done

**Review by hellofadude — APPROVED** [review]

**Review by DawidGrzeszczak — APPROVED** [review]
good job

File: src/test/java/com/kainos/ea/dao/SpecificationLevelTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.sql.SQLException;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hibernate.validator.internal.util.Contracts.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertSame;
+
+@ExtendWith(MockitoExtension.class)
```
Comment (hellofadude) [inline src/test/java/com/kainos/ea/dao/SpecificationLevelTest.java:14]: Remove mockito as you're not mocking anything here

Comment (KewinTaoAnh) [reply src/test/java/com/kainos/ea/dao/SpecificationLevelTest.java]: done

**Pull request merged**